Headers/species_database_mod.F90

```diff
@@ -508,7 +508,7 @@ SUBROUTINE Init_Species_Database( Input_Opt, SpcData, SpcCount, RC )
           ELSE IF ( INDEX( key, "%Is_DryDep" ) > 0 ) THEN
              CALL QFYAML_Add_Get( yml, key, v_bool, "", RC )
              IF ( RC /= GC_SUCCESS ) GOTO 999
-             IF ( v_bool ) THEN
+             IF ( v_bool .AND. ThisSpc%Is_Advected ) THEN
```

Comment:
I added this to avoid a seg fault when deallocating State_Chm%DryDepVel during Finalize.